pkg/kev/converter/kubernetes/utils.go

```diff
@@ -950,3 +952,31 @@ func volumeByNameAndFormat(name string, formatter func(string) string, volumes c
 	}
 	return composego.VolumeConfig{}
 }
+
+// hasDefaultIngressBackendKeyword determines whether the host value list contains the keyword used to create
+// a default backend ingress.
+func hasDefaultIngressBackendKeyword(v []string) bool {
+	return strings.Contains(strings.Join(v, ""), DefaultIngressBackendKeyword)
```

Comment:
should we also strip whitespaces here or is it done by the config object already?

Reply:
Good catch!

Will trim space in the `p.SvcK8sConfig.Service.Expose.Domain` value in `exposeService()` before it's returned.

Reply:
I think it'd make sense to do it in this PR

Reply:
@marcinc The whitespace is stripped upstream - there's no extra whitespace at this point.

But maybe I missing the point?